dds_web/api/project.py

```diff
@@ -298,3 +300,70 @@ def put(self):
                 break
 
         return flask.jsonify({"updated": updated, "error": error, "tries": current_try})
+
+
+class CreateProject(flask_restful.Resource):
+    @auth.login_required
```

Comment:
You can set this to 
`@auth.login_required(role="admin")`
for now

Reply:
Done!